roles/common/templates/etc_apache2_ssl.conf.j2

```diff
@@ -2,13 +2,13 @@ SSLEngine on
 SSLProtocol ALL -SSLv2 -SSLv3
 SSLHonorCipherOrder On
 SSLCompression off
+SSLCertificateKeyFile	/etc/letsencrypt/live/{{ domain }}/privkey.pem
+SSLCertificateFile	/etc/letsencrypt/live/{{ domain }}/cert.pem
+SSLCertificateChainFile	/etc/letsencrypt/live/{{ domain }}/chain.pem
```

Comment:
Mobile Safari will complain about the certificate unless we use fullchain.pem here. See also: https://community.letsencrypt.org/t/untrusted-on-mobile-newbe/7903